packages/tests/src/commonlib/frontendValidator.ts

```diff
@@ -127,6 +127,7 @@ export class FrontendValidator {
     fs.access(indexPath, fs.constants.F_OK, (err) => {
       chai.assert.isNull(err);
     });
+    return Promise.resolve();
```

Comment:
It appears that some functions are returning void while others are returning Promise.resolve(). What's the distinction between the two? Could we make them consistent?

Reply:
It depends on the function declaration.
If the function declaration has the async keyword, we should return promise resolve, else return void.
There will be an eslint promise rule to check it.